synapse_data_warehouse/synapse_aggregate/dynamic_tables/V2.50.0__unique_user_uploads_aggregate.sql

```diff
@@ -0,0 +1,104 @@
+USE SCHEMA {{database_name}}.synapse_aggregate; --noqa: JJ01,PRS,TMP
+
+CREATE OR REPLACE DYNAMIC TABLE USER_UPLOADS
+    (
+	    AGG_PERIOD VARCHAR(16777216) COMMENT 'The dimension of the aggregate (e.g., YEARLY, MONTHLY, DAILY). Part of composite PK.',
+	    USER_COUNT INT COMMENT 'The number of distinct unique users uploading during the aggregation period.',
+	    AGG_YEAR NUMBER(38,0) COMMENT 'Year of the aggregation period.',
```

Comment:
nit: Consistent tabbing with the next few lines

Reply:
probably a tab vs. spaces thing 😄 If you delete the indent for all these lines and re-do the indent, that ought to fix it